### PR TITLE
[SID-1199] Add the otp_via_email auth method

### PR DESCRIPTION
**Pull request by ikovic**
## Description

[YouTrack issue](https://todo.irdcorp.dev/issue/SID-1199)

Follow up work to get the `otp_via_email` method working.

## Checklist

- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have generated the new version of the docs website and smoke tested it
- [ ] I have checked that my changes haven't caused semantic errors in the existing docs
- [ ] I have updated the README and DEVELOPMENT files

### CODE REVIEW

File: packages/react/src/domain/handles.ts [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
 import { Factor } from "@slashid/slashid";
 import {
   getList,
   findFlagByDialCode,
 } from "country-list-with-dial-code-and-flag";
 import { TextConfigKey } from "../components/text/constants";
 
-const FACTORS_WITH_EMAIL = ["webauthn", "email_link"];
+const FACTORS_WITH_EMAIL = ["webauthn", "otp_via_email", "email_link"];
 const FACTORS_WITH_PHONE = ["otp_via_sms", "sms_link"];
```
Comment (paulo-raca) [inline packages/react/src/domain/handles.ts:10]: :thinking: IIRC we can do WebAuthn + SMS

**Review by jake-slashid — APPROVED** [review]

File: packages/react/src/dev.tsx [hunk 2]
```diff
@@ -157,8 +173,14 @@ ReactDOM.createRoot(document.getElementById("root") as HTMLElement).render(
     >
       <div className="layout">
         <div>
-          <h2>Switch to default org</h2>
-          <Config />
+          <div>
```
Comment (paulo-raca) [inline packages/react/src/dev.tsx:178]: Is this a test page?

**Review by paulo-raca — APPROVED** [review]
LGTM, thank you

**Review by balenio — APPROVED** [review]

File: packages/react/src/dev.tsx (hunk 2 above)
Comment (ikovic) [reply packages/react/src/dev.tsx:178]: Correct, this is our development page where we render all the different app configs. Not exposed to customers in any way 😄

File: packages/react/src/domain/handles.ts (hunk 1 above)
Comment (ikovic) [reply packages/react/src/domain/handles.ts:10]: I think we can but we don't support it for some reason I cannot remember right now 😓

**Pull request merged**